pkg/replication/replication.go

```diff
@@ -296,6 +306,16 @@ func (c *Config) EditRule(opts Options) error {
 		}
 	}
 
+	if opts.ReplicateSync != "" {
+		switch opts.ReplicateSync {
+		case ReplicateSync:
+			c.ReplicateSync = true
```

Comment:
```suggestion
			c.ReplicateSync = opts.ReplicateSync == ReplicateSync
```

can be written in one line? and validation for `ReplicateSync` value to be outside of this check?

Reply:
as mentioned above,the ReplicateSync setting is at bucket level rather than rule level, hence this check.